playwright.config.ts

```diff
@@ -0,0 +1,32 @@
+import { devices, PlaywrightTestConfig } from '@playwright/test';
+
+// Config file docs: https://playwright.dev/docs/test-configuration
+const config: PlaywrightTestConfig = {
+  webServer: {
+    command: 'pnpm start',
+    port: 3000,
+    reuseExistingServer: !process.env.CI,
+  },
+  testIgnore: '**/util/__tests__/**',
+  retries: process.env.CI ? 2 : 0,
+  reporter: process.env.CI
+    ? 'list'
+    : [['html', { outputFolder: 'playwright/report/' }]],
+  outputDir: 'playwright/test-results/',
```

Comment:
The output directory for files created during test execution. Default is `<package.json-directory>/test-results`
https://playwright.dev/docs/api/class-testconfig#test-config-output-dir